.github/workflows/ci-17.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and Test
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  build:
+
+    name: Build and Test (17)
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Java 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: 11
+
+      - name: Build 
+        run: |
+          ./gradlew build --build-cache
+
+      - name: Setup Java 17
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
```

Comment:
May be a very dumb question. I dont understand Java distributions yet, but can we use with what with support on OpenSearch for consistency? 
We use adoptium.

Ref: https://github.com/opensearch-project/OpenSearch/pull/1358

Reply:
Temurin is a distribution of adoptium - https://adoptium.net/